.eslintrc

```diff
@@ -10,15 +10,15 @@
   "parserOptions": {
     "project": "./tsconfig.json"
   },
-  "plugins": [
-    "@typescript-eslint"
-  ],
+  "plugins": ["@typescript-eslint"],
   "rules": {
-    "@typescript-eslint/indent": "off"
+    "@typescript-eslint/indent": "off",
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "react/prop-types": "off"
```

Comment:
Given that we could use an interface to declare the props of a component I think we don't need this to be mandatory. Anyways, If we need prop-types in the generated js bundles maybe we could try something like https://www.npmjs.com/package/babel-plugin-typescript-to-proptypes

Reply:
I don't think we need prop-types either 👍